ext/mruby_engine/ext.c

```diff
@@ -273,6 +276,15 @@ static VALUE ext_mruby_engine_stat(VALUE rself) {
   uint64_t memory_count = me_mruby_engine_get_memory_count(self);
   rb_hash_aset(stat, ID2SYM(me_ext_id_memory), ULONG2NUM(memory_count));
 
+  int64_t ctx_switches_v = me_mruby_engine_get_ctx_switches_voluntary(self);
+  rb_hash_aset(stat, ID2SYM(me_ext_id_ctx_switch_v), LONG2NUM(ctx_switches_v));
```

Comment:
Whatever you guys think is best... The value of having at least `-1` have a special semantic is being able to test for these... It could well be `0` is the actual value after some script ran... especially tiny ones. `0` certainly will be a value post running script for voluntary context switches...